lib/system.g

```diff
@@ -485,6 +486,19 @@ CallAndInstallPostRestore( function()
        "\n" );
       QuitGap();
     fi;
+
+    # Evaluate the '--print-gaproot' option.
+    if GAPInfo.CommandLineOptions.( "print-gaproot" ) then
+      for line in GAPInfo.RootPaths do
+        value:= SHALLOW_COPY_OBJ( line );
+        APPEND_LIST_INTR( value, "sysinfo.gap" );
+        if IsExistingFile( value ) then
+          PRINT_TO( "*errout*", line );
```

Comment:
```suggestion
          PRINT_TO( "*stdout*", line );
```

Reply:
I would print the --help output to stdout, too, yes. I didn't (consciously) realize that we print it to stderr.

However, I don't care as much in this case: `--print-gaproot` is exclusively meant for use by scripts and so having it print to stderr would be rather inconvenient.

In contrast, I don't have to process the output of `--help` from a script, so I care less. (I am not claiming nobody has need for that, just that I don't -- and before I'd parse that, I'd rather write code which prints the information in there in a machine readable format. But that's another story).

(Actually, as I write this, I realize that I might want to parse the output `gap --help` after all: to efficiently figure out if `--print-gaproot` is supported 😂 . But in practice, I think I'd rather use an invocation like `gap --print-gaproot -q -b -A -r path/to/fallback/code.g`

Reply:
tl;dr: sure, let's change --help to print to stdout, too, but perhaps in another PR?